contracts/payloads/IGP105/PayloadIGP105.sol

```diff
@@ -0,0 +1,585 @@
+                rateAtUtilizationKink2: 4 * 1e2, // 4%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV2s(params_);
+        }
+    }
+
+    // @notice Action 3: Make Multisig 2 as deployer on all factories
+    function action3() internal isActionSkippable(3) {
+        // Set TEAM_MULTISIG_2 as deployer on all factories
+        IFluidVaultFactory(VAULT_FACTORY).setFactoryAuth(TEAM_MULTISIG_2, true);
+        IFluidDexFactory(DEX_FACTORY).setFactoryAuth(TEAM_MULTISIG_2, true);
+        IFluidLendingFactory(LENDING_FACTORY).setFactoryAuth(
+            TEAM_MULTISIG_2,
+            true
+        );
+        ISmartLendingFactory(SMART_LENDING_FACTORY).setFactoryAuth(
+            TEAM_MULTISIG_2,
+            true
+        );
+    }
+
+    // @notice Action 4: Set wstUSR launch limits and dust limits
+    function action4() internal isActionSkippable(4) {
+        // Set launch limits for existing WSTUSR/STABLE vaults (T1 vaults)
+        {
+            address wstUSR_USDC_VAULT = getVaultAddress(110);
+            
+            // [TYPE 1] WSTUSR/USDC vault - Launch limits
+            VaultConfig memory VAULT_wstUSR_USDC = VaultConfig({
+                vault: wstUSR_USDC_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDC_ADDRESS,
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 12_000_000 // $12M
+            });
+            
+            setVaultLimits(VAULT_wstUSR_USDC);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDC_VAULT, TEAM_MULTISIG, false);
+        }
+        
+        {
+            address wstUSR_USDT_VAULT = getVaultAddress(111);
+            
+            // [TYPE 1] WSTUSR/USDT vault - Launch limits
+            VaultConfig memory VAULT_wstUSR_USDT = VaultConfig({
+                vault: wstUSR_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 12_000_000 // $12M
+            });
+            
+            setVaultLimits(VAULT_wstUSR_USDT);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDT_VAULT, TEAM_MULTISIG, false);
+        }
+        
+        {
+            address wstUSR_GHO_VAULT = getVaultAddress(112);
+            
+            // [TYPE 1] WSTUSR/GHO vault - Dust limits
+            VaultConfig memory VAULT_wstUSR_GHO = VaultConfig({
+                vault: wstUSR_GHO_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: GHO_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
```

Comment:
this gets launch limits too and remove MS auth

Reply:
<img width="993" height="42" alt="Screenshot 2025-07-26 at 8 35 59 PM" src="https://github.com/user-attachments/assets/541c6b36-f9ad-47e6-9669-b0dd60809b6e" />
ohh i read it from here